base/applications/notepad/settings.c

```diff
@@ -100,9 +101,54 @@ static BOOL QueryBool(HKEY hKey, LPCTSTR pszValueName, BOOL *pbResult)
     return TRUE;
 }
 
-static BOOL QueryString(HKEY hKey, LPCTSTR pszValueName, LPTSTR pszResult, DWORD dwResultSize)
+static BOOL QueryString(HKEY hKey, LPCTSTR pszValueName, LPTSTR pszResult, DWORD cchResult)
```

Comment:
Why changing/breaking the hungarian notation?

Reply:
http://www.imm.dtu.dk/~alan/hungarian.html
`cch` is also in Hungarian notation.
Renamed `cchResult` as `dwResultLen`.

Reply:
There is a confusion between string size and length. I want to use the word 'length' (or 'len') to specify the length of a string.

Reply:
Using dwResultLength should be fine though, if you think it is more descriptive. I just find dw for DWORD much more common and logical here than cch

Reply:
OK.

Reply:
OK.

Reply:
cch is an actual useful notation that perfectly communicates the actual intent, as opposed to dw, which does not add anything of value.